lib/error.g

```diff
@@ -204,7 +204,7 @@ BIND_GLOBAL("ErrorInner",
     if printThisStatement then 
         if context <> GetBottomLVars() then
             PrintTo("*errout*"," in\n  \c");
-            PRINT_CURRENT_STATEMENT(context);
+            PRINT_CURRENT_STATEMENT("*errout*", context);
             Print("\c");
```

Comment:
Shouldn't this also be printed to `*errout*`?

Reply:
Ofc. Thanks :)